pandas/tests/test_base.py

```diff
@@ -1074,9 +1074,10 @@ class TestToIterable(object):
         'method',
         [
             lambda x: x.tolist(),
+            lambda x: x.to_list(),
             lambda x: list(x),
             lambda x: list(x.__iter__()),
```

Comment:
you will need to run the test and then catch any instance of .tolist as it will show a warning

Reply:
Hm. I've run the tests (via ``pytest pandas/tests/test_base.py -v -k TestToIterable`` and I don't get any warnings there.

Reply:
pls run the test suite itself. likely this is used elsewhere which need catching.

Reply:
It seems that I can't produce any warning related to these changes. I also quick-checked the Travis logs and compared them with other builds.